content/docs/tina-init-tutorial.md

```diff
@@ -0,0 +1,221 @@
+Hello! and thanks for bootstrapping a Tina App! Before you do anything click on "toggle edit state" button and a pencil icon in the bottom left hand corner will appear. You can now edit this content in real time! Click save and notice that you have update the Hello world blog post in the local file system.
+
+## Next steps
+
+### Wrap your App in "Edit State" (**Can probably skip this as CLI did this for you**)
```

Comment:
Maybe we can remove the () part and place a quick paragraph that explains if you bootstrapped the application we have taken care of it for you.